sensu/server.sls

```diff
@@ -187,6 +188,14 @@ uchiwa:
 {{ logship('sensu-api.log',  '/var/log/sensu/sensu-api.log', 'sensu', ['sensu', 'sensu-api', 'log'],  'rawjson') }}
 {{ logship('uchiwa.log',  '/var/log/upstart/uchiwa.log', 'sensu', ['sensu', 'sensu-dashboard', 'uchiwa', 'log'],  'rawjson') }}
 
+/etc/collectd/collectd.conf.d/processes-redis.conf:
+  file.managed:
+    - source: salt://sensu/files/graph-redis.conf
```

Comment:
Minor thing but could  we rename this to `collectd_conf_process_redis.conf` so that we can look at the files/ dir and have an idea where each file is used for.